Mage/src/main/java/mage/game/permanent/token/TarmogoyfToken.java

```diff
@@ -0,0 +1,42 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.CardTypesInGraveyardCount;
+import mage.abilities.effects.common.continuous.SetBasePowerToughnessPlusOneSourceEffect;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.Zone;
+
+/**
+ * @author LoneFox
+ */
+public final class TarmogoyfToken extends TokenImpl {
+
+    private static final DynamicValue powerValue = CardTypesInGraveyardCount.ALL;
+
+    public TarmogoyfToken() {
+        super("Tarmogoyf Token",
+                "Tarmogoyf’s power is equal to the number of card types among cards in all graveyards and its toughness is equal to that number plus 1.");
```

Comment:
You used wrong `’` instead of `'`

Reply:
and actually, I'm not sure that whole text is meant to be there.

Reply:
Ooh, I wasn't sure either but I though I should make it look like the token on scryfall

Reply:
For `'`, Scryfall is not using that punctuation symbol, so you need to make sure you fix it if you copy/paste text from there.

The second String entry is meant to be what's written on the card creating the token.
Here it should have been (I've fixed it in master) "Tarmogoyf token" so it can be used by text generation of `CreateTokenEffect`.